### DISCUSSION TITLE
< Request Features > [ frame interpolation ] similar to nvidia frame generation

### DISCUSSION THREAD

**Discussion by humanana** [Blinue/Magpie#498] [Ideas]
is it possible to implement something like frame interpolation in the magpie such as filter? to increase fps?

something like this https://www.youtube.com/watch?v=882c25af2hM
https://www.svp-team.com/

instead only on vdo player but can use with gaming also? 

similar to nvidia frame generation

**Comment by Blinue** [comment]
Thank you for the suggestion. Interpolation requires knowledge of both the previous and next frame, which is not possible in real-time scaling. Some degree of "prediction" is indeed possible, but this feature is not planned as it is too complex and Magpie's main focus is not on improving game performance.

**Reply by humanana** [reply]
alright, thank you for having a look into the idea :>

**Comment by ThreeDeeJay** [comment]
👀 https://steamcommunity.com/app/993090/discussions/0/4039232337479089112

**Comment by humanana** [comment]
> 👀 https://steamcommunity.com/app/993090/discussions/0/4039232337479089112

thank you I'll test it +w+
